### PR TITLE
[WFLY-13706] Add documentation describing how to use the Keycloak adapters with WildFly Elytron.

### PR DESCRIPTION
**Pull request by darranl**
https://issues.redhat.com/browse/WFLY-13706

### CODE REVIEW

**Comment by darranl** [comment]
I have added the 22.x label as it would be good to get this published so we can start to draw attention to it as a new way to install the Keycloak adapters.

File: docs/src/main/asciidoc/_elytron/Keycloak_Integration.adoc [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+----
+<plugin>
+    <groupId>org.wildfly.plugins</groupId>
+    <artifactId>wildfly-jar-maven-plugin</artifactId>
+    <version>${version.wildfly.jar.maven.plugin}</version>
+    <configuration>
+        <feature-packs>
+            <feature-pack>
+                <location>wildfly@maven(org.jboss.universe:community-universe):current</location>
+                <inherit-configs>false</inherit-configs>
```
Comment (jfdenise) [inline docs/src/main/asciidoc/_elytron/Keycloak_Integration.adoc:69]: inherit-configs and inherit-packages shouldn't be mentioned.

File: docs/src/main/asciidoc/_elytron/Keycloak_Integration.adoc [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                <inherit-packages>false</inherit-packages>
+            </feature-pack>
+            <feature-pack>
+                <groupId>org.keycloak</groupId>
+                <artifactId>keycloak-adapter-galleon-pack</artifactId>
+                <version>12.0.1</version>
+                <inherit-configs>false</inherit-configs>
```
Comment (jfdenise) [inline docs/src/main/asciidoc/_elytron/Keycloak_Integration.adoc:76]: inherit-configs and inherit-packages shouldn't be mentioned.

**Review by jfdenise — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/_elytron/Keycloak_Integration.adoc [hunk 3]
```diff
@@ -0,0 +1,125 @@
+[[Keycloak_Integration]]
+= Using Keycloak with WildFly Elytron
+
+When securing deployments using a Keycloak server a set of client side adapters provided by Keycloak need to be 
+installed to WildFly including a new subsystem to support configring the client side adapters.  Depending on how
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_elytron/Keycloak_Integration.adoc:5]: s/configring/configuring

**Review by bstansberry — COMMENTED** [review]
Other than Jean-Francois' comment and one minor typo I noticed, this LGTM so please feel free to merge when those are addressed.

**Comment by darranl** [comment]
Superseded by https://github.com/wildfly/wildfly/pull/13849

**Pull request closed without merging**